common/config/rush/command-line.json

```diff
@@ -180,6 +188,20 @@
       "description": "flag to enable coverage collection for jest",
       "associatedCommands": ["test", "test-ci"]
     },
+    {
+      "parameterKind": "flag",
+      "longName": "--debug",
+      "shortName": "-d",
+      "description": "enables logging for rush clean",
```

Comment:
We might use this flag for other commands so just say `enables debug logging for associated command`. Can add this as a flag for `start-server` as well since `react-static` has a --debug flag.